### PR TITLE
Adding a page that explains a bit the Batch Explorer as a tool.

### PR DESCRIPTION
**Pull request by mmilenkovic**

### CODE REVIEW

**Review by druiz0992 — CHANGES_REQUESTED** [review]
I think text is fine, but some pictures would greatly help to clarify. Is it possible to embed some captures of the block explorer?

Another thing: I did not see this page linked to _sidebar.md. I would just add it after SDK and API inside the developers section.  I would postpone the creation of a new Tools section until we have published the wallet and other tools. Otherwise it is a bit poor (this is just an opinion, and I am fine either way).

File: docs/tools/batch-explorer.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+## Coordinator
+By searching for the Ethereum address of an existing coordinator, or by navigating through the app the user lands on the page listing details of that coordinator. Those details include:
+* forger and withdrawal addresses
+* full list of forged batches
+* full list of winner bids (including slot, total bids and bid amount)
+
+## Searching through Hermez Batch Explorer
+There are limited options available for search and navigation in the app itself. Those include:
+* Ethereum address
+* hez Ethereum address
```
Comment (jleebee) [inline docs/tools/batch-explorer.md]: HEZ Ethereum address

Comment (mmilenkovic) [reply docs/tools/batch-explorer.md]: Done.

**Review by davidsrz — DISMISSED** [review]

**Review by druiz0992 — APPROVED** [review]
Thanks

**Comment by druiz0992** [comment]
Already reviewed by three people

**Pull request merged**